person.rb

```diff
@@ -18,4 +22,12 @@ def can_use_services?
     of_age? || @parent_permission
   end
   public :can_use_services?
+
+  def correct_name
+    @name
+  end
+
+  def add_rental(date, book)
+    @rentals.push(Rental.new(date, book))
+  end
```

Comment:
- [ ] Rental class constructor also expects `person` as a parameter so please make sure to pass the `person` parameter in the Rental class constructor. You can consider the below code.

```ruby
  def add_rental(date, book)
    @rentals.push(Rental.new(date,self, book))
  end